cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/JoinDecider.scala

```diff
@@ -24,6 +24,8 @@ trait JoinDecider {
    */
   def decide(info: SeedNodesInformation): Future[JoinDecision]
 
+  // FIXME API support implementations in Java, returning CompletionStage?
```

Comment:
Low priority I think as not sure anyone will implement a JoinDecider

Reply:
ok, I agree